server/app.js

```diff
@@ -0,0 +1,26 @@
+
+const express = require('express');
+const path = require('path');
+
+
+const favicon = require('serve-favicon');
+const bodyParser = require('body-parser');
+const compression = require('compression');
+
+const controllers = require('./controllers/router');
+
+const app = express();
+
+app.disable('x-powered-by');
+app.use(compression());
+// app.use(favicon(path.join(__dirname, '..', 'public', 'assets', 'favicon.ico')));
+app.use(bodyParser.json());
```

Comment:
We didn't teach you this but in the recent versions of express, you don't need to use body parser. Instead, you can use the built-in express functionality like this: 
```
  app.use(express.json());
```